src/components/CheckEditor/CheckEditorNew.test.tsx

```diff
@@ -22,6 +23,11 @@ jest.mock('hooks/useAlerts', () => ({
   }),
 }));
 
+jest.mock('@grafana/ui', () => ({
+  ...jest.requireActual('@grafana/ui'),
+  CodeEditor: () => <textarea />,
+}));
```

Comment:
Similar to the reasoning above, I'd do this globally similar to how we mock `@grafana/runtime`.